.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We have a standard ESLint file in https://github.com/netzstrategen/dotfiles. Please use it. 🙏

Reply:
P.S. Pretty sure you had to adjust code style after apply the proper ESLint config. e.g. I see lot of missing spaces after `function` keyword.